index.html

```diff
@@ -1445,6 +1645,50 @@ <h2 id="AudioContext">
         </p>
         <dl title='interface AudioContext : BaseAudioContext' class='idl'>
           <dt>
+            Constructor
+          </dt>
+          <dd>
+            <p>
+              <span class="synchronous">When creating an <a>AudioContext</a>,
+              execute these steps:</span>
+            </p>
+            <ol>
+              <li>Set a <code>control thread state</code> to
+              <code>suspended</code> on the <a>AudioContext</a>.
+              </li>
+              <li>Send a <a>control message</a> to start processing.
+              </li>
+            </ol>
+            <p>
+              Sending a <a>control message</a> to start processing means
+              executing the following steps:
+            </p>
+            <ol>
+              <li>Attempt to <a href="#acquiring">Acquire system resources</a>.
+              </li>
+              <li>In case of failure, abort these steps.
```

Comment:
See the `<p class="note">` below, we can't throw here because opening an audio channel is async, but UA can warn the user.

Maybe Chrome can throw because it has an hard-coded limit ?